Add Slack apps for developers (#2391)
sindresorhus/awesome: readme.md

## Contents

> - [Contents](#contents)

Author: ```suggestion
```

- [Slack](https://github.com/matiassingers/awesome-slack#readme) - Team collaboration.
	- [Communities](https://github.com/filipelinhares/awesome-slack#readme)

> 	- [Slack Apps for developers](https://github.com/axolo-co/awesome-slack-apps)

Author: ```suggestion
	- [Slack Apps for developers](https://github.com/axolo-co/awesome-slack-apps) - List of Slack integration to help developers and tech teams by more productive.
```

> 	- [Slack Apps for developers](https://github.com/axolo-co/awesome-slack-apps) - List of Slack integration to help developers and tech teams by more productive.

Author: ```suggestion
	- [Slack Apps For Developers](https://github.com/axolo-co/awesome-slack-apps) - List of Slack integration to help developers and tech teams by more productive.
```

> 	- [Slack Apps For Developers](https://github.com/axolo-co/awesome-slack-apps) - List of Slack integration to help developers and tech teams by more productive.

Author: ```suggestion
	- [Slack Apps For Developers](https://github.com/axolo-co/awesome-slack-apps) - List of Slack integration to help developers and tech teams be more productive.
```

> 	- [Slack Apps For Developers](https://github.com/axolo-co/awesome-slack-apps) - List of Slack integration to help developers and tech teams be more productive.

Reviewer: Contribution guide says to link directly to the README, and the tagline should be a description of the topic, not a description of the list.
```suggestion
	- [Slack Apps For Developers](https://github.com/axolo-co/awesome-slack-apps#readme) - Slack integrations to help developers and tech teams be more productive.
```

> 	- [Slack Apps For Developers](https://github.com/axolo-co/awesome-slack-apps#readme) - Slack integrations to help developers and tech teams be more productive.

Reviewer 2: The title here should be the same as the title of your list.

Reviewer 2: Same with the PR title.

> 	- [Slack Apps For Developers](https://github.com/axolo-co/awesome-slack-apps#readme) - Slack integrations to help developers and tech teams be more productive.

Author: ```suggestion
	- [Slack Apps For Developers](https://github.com/axolo-co/awesome-slack-apps-for-developers#readme) - Slack integrations to help developers and tech teams be more productive.
```

> 	- [Slack Apps For Developers](https://github.com/axolo-co/awesome-slack-apps-for-developers#readme) - Slack integrations to help developers and tech teams be more productive.

Reviewer 2: ```suggestion
	- [Slack Apps for Developers](https://github.com/axolo-co/awesome-slack-apps-for-developers#readme) - Slack integrations to help developers and tech teams be more productive.
```

And the list heading should match this capitalisation.

Reviewer 3: Jo
